launch/handeye_file.launch

```diff
@@ -43,7 +43,6 @@
     <!-- When you load transforms you saved to a file earlier, this is where the file is saved -->
     <param name="transform_pairs_load_filename"  type="str" value="$(arg data_folder)/$(arg filename)" />
     <param name="output_calibrated_transform_filename" type="str" value="$(arg data_folder)/$(arg calibrated_filename)" />
-    <param name="load_transforms_from_file" type="bool" value="true"/>
```

Comment:
why is this parameter removed? we utilize it https://github.com/jhu-lcsr/handeye_calib_camodocal/blob/565b818c5df83cf4431eedcd1dd5012d57560eb4/src/handeye_calibration.cpp#L314

Reply:
I removed it because I assume that this launch file address specifically the case where the input paris transforms already live on the disk. The param is passed and set to True to the ros param server

```
<param name="load_transforms_from_file" type="bool" value="true"/>
```
If it is not the case, Ill put it back :)

Reply:
we prefer to keep the variables consistent so it isn't necessary to also look at the code, if that's ok.